src/components/tx/DecodedTx/index.tsx

```diff
@@ -119,63 +89,34 @@ const DecodedTx = ({
             expandIcon={<ExpandMoreIcon />}
             className={accordionCss.accordion}
           >
-            <span style={{ flex: 1 }}>Transaction details</span>
-
-            {decodedData
-              ? decodedData.method
-              : tx?.data.operation === OperationType.DelegateCall
-              ? 'Delegate call'
```

Comment:
This information is gone now. 
I think we should always display if an operation is a Delegate call. We should also display the Delegate Call warning here, if `txDetails?.txData?.trustedDelegateCallTarget` is false.

Currently we only display such warning in `SingleTxDecoded`. But there were cases were scammers would try to bait you into a high level delegateCall - which can be submitted through other interfaces / phishing sites to our tx service.